framework/src/controller/event.ts

```diff
@@ -22,7 +22,7 @@ export interface EventInfoObject {
 	readonly data: object;
 }
 
-export type EventCallback = (action: EventInfoObject) => void;
+export type EventCallback = (action: EventInfoObject) => void | Promise<void>;
```

Comment:
why the type here is changed?

Reply:
To register a call back function with async. 

```js
channel.once('app:ready', async () => {
```